fineract-doc/src/docs/en/chapters/security/oauth.adoc

```diff
@@ -51,6 +51,8 @@ curl --location --request POST \
 --data-urlencode 'client_secret=<enter the client secret from credentials tab>'
 ----
 
+Before making calls to the fineract api, ensure that you set the property spring.security.oauth2.resourceserver.jwt.issuer-uri within application.properties to the issuer url of the keyclock server, which generated the token. After changing the property, you may have to rebuild. Alternatively, you can expose a docker variable through that file, so you could dynamically pass new issuer url to the docker image.
```

Comment:
IMO this property is already exposed as an environment variable: 

https://github.com/apache/fineract/blob/4e877451c32621411c4973c3d30c1dda6e104a33/fineract-provider/src/main/resources/application.properties#L251-L252

`FINERACT_SERVER_OAUTH_RESOURCE_URL`